hoot-core/src/main/cpp/hoot/core/schema/MetadataTags.h

```diff
@@ -111,6 +111,12 @@ class MetadataTags
   inline static const QString HootScoreReview()         { return "hoot:score:review"; }
   inline static const QString HootScoreUuid()           { return "hoot:score:uuid"; }
 
+  // used with JOSM validation interop; match those used in JosmValidator.java
```

Comment:
names of tags that get added either during validation or during cleaning with validation tags enabled